src/main/java/de/retest/recheck/cli/subcommands/Version.java

```diff
@@ -10,7 +10,7 @@
 @Command( name = "version", description = Version.VERSIOND_COMMAND_DESCRIPTION )
 public class Version implements Runnable {
 
-	public static final String VERSIOND_COMMAND_DESCRIPTION = "Display version info.";
+	public static final String VERSIOND_COMMAND_DESCRIPTION = "Display version info of this CLI.";
```

Comment:
Hmmm, actually it displays version info for all the stuff returned by `VersionProvider`, which currently is the CLI and Java version. However, this may change in the future (we might also include the used recheck version), so maybe something more generic?